presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddExchanges.java

```diff
@@ -153,41 +148,29 @@ public PlanNode optimize(PlanNode plan, Session session, Map<Symbol, Type> types
         private final PreferredProperties preferredProperties;
```

Comment:
I left it in in case some new operator was added that was problematic so people would know to be careful in AddExchanges.  But the comment doesn't belong here.  And it's probably sufficient that there's a comment in EnforceRequiredJoinMethods